code/game/objects/items/storage/belt.dm

```diff
@@ -151,7 +151,7 @@
 	storage_slots = 14
 	max_w_class = SIZE_MEDIUM
 	max_storage_space = 28
-	var/mode = 0 //Pill picking mode
+	var/mode = 1 //Picking from pill bottle mode
```

Comment:
You should probably be using defines, for example, there is a define here called SIZE_MEDIUM. https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#no-magic-numbers-or-strings

Reply:
meant for this, max_storage_space = 28, storage_slots = 14, It's not even his code, hence it's a bit of an ask.

Reply:
And aren't true and false just defines for 0 and 1? Might be wrong.

Reply:
not saying you need to do this, but this is an example of how boolean defines are properly done 
<img width="438" alt="Screen Shot 2023-10-09 at 10 11 02 PM" src="https://github.com/cmss13-devs/cmss13/assets/122310258/49b98490-253d-4837-a554-eeaa6dc3e1ce">
It would eliminate the need for you to place a comment because it would self-explanatory what the define is.